core/model/src/main/java/com/merxury/blocker/core/model/data/UserEditableSettings.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 Blocker
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.merxury.blocker.core.model.data
+
+import com.merxury.blocker.core.model.data.ControllerType.IFW
+import com.merxury.blocker.core.model.preference.DarkThemeConfig
+import com.merxury.blocker.core.model.preference.RuleServerProvider
+import com.merxury.blocker.core.model.preference.RuleServerProvider.GITHUB
+
+data class UserEditableSettings(
```

Comment:
This data class should be in the `:feature:settings module. Why did you move this class to `:core:model`?

Reply:
Because it needs to be used in userEditableSettingsTestData in core:testing